### PR TITLE
Camera setup

### PR DESCRIPTION
**Pull request by Jarr1t**
Some basic setup for gaining permissions to camera and gallery

### CODE REVIEW

**Review by antsanchez198 — APPROVED** [review]
Looks good!

File: components/.Home.js.swp
Comment (Ricardo-21) [inline components/.Home.js.swp:1]: Is this file important to the camera access setup?

File: components/Home.js [hunk 1]
```diff
@@ -1,15 +1,40 @@
...
+            <Button 
+            title="Use Camera"
+            onPress={() => toggleCamera()}
+            />
+        </View>
     </View>
+    )
+};
+
+const styles = StyleSheet.create({
```
Comment (Ricardo-21) [inline components/Home.js:33]: We may want to start creating style sheets in another directory as @e-mbrown stated #12

**Review by Ricardo-21 — APPROVED** [review]
Overall LGTM!

**Pull request merged**